packages/andromeda-socket/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "andromeda-socket"
+version = "1.0.0"
```

Comment:
The contracts using this package have their version set at `0.1.0`. Doesn't seem consistent